pkg/process/debug.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package process
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"net/http/pprof"
+
+	"go.uber.org/zap"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+	"gopkg.in/spacemonkeygo/monkit.v2/present"
+)
+
+func init() {
+	// zero out the http.DefaultServeMux net/http/pprof so unhelpfully
+	// side-effected.
+	*http.DefaultServeMux = http.ServeMux{}
+}
+
+func initDebug(ctx context.Context, logger *zap.Logger, r *monkit.Registry) (
```

Comment:
I don't think libraries should log

Reply:
i'm not sure i agree, but regardless, this library is a pretty unique library. it's setting up process-wide information.